src/__tests__/GoalParser.test.ts

```diff
@@ -70,3 +77,18 @@ describe("getTime function", () => {
     expect(timeSuggestion()).toEqual(time());
   });
 });
+function urlDetection() {
+  if (goalUrl.title.search(urlDetector) !== -1) {
+    return "Link";
+  }
+
+  return {
+    urlData,
+  };
+}
+describe("getUrl function", () => {
+  it("should return link : Link", () => {
+    const a = "Link";
+    expect(urlDetection()).toEqual(a);
```

Comment:
```suggestion
    expect(urlDetection()).toEqual("Link");
```
This improves readability.